### ISSUE TITLE
Send automatic email when a user signed up for a task with an ical file

### ISSUE THREAD

**Issue by Doca** [coders4help/volunteer_planner#475]
**Is your idea or feature request related to a problem? Please describe.**
**As a volunteer** I want to have a confirmation for which shift and task I signed up. Then I don't need to log into volunteer-planner again if I want to look up the place and date again. The ical entry helps me to not forget my shift.
In the email the following Information would be helpful:
- Facilityname
- Taskname
- Place Address
- Shift starting_time and ending_time

**Describe the solution you'd like**
The system should send an Email to the volunteer after signing up for a task. In the email there should be a ical file for easy adding it to a calendar.

**Email Text Draft**
Hello {{user}},

You just signed up for the following task: {{Taskname}}.
Your shift begins at {{starting_time}} and ends at {{ending_time}}.
The facility {{facility name}} you will find at the following address:

{{ Address of the place}}

If you can't make it please unsubscribe your task at the volunteer-planner.org
Your Volunteer-Planner Team

**Comment by pitpalme** [comment]
Message text should be fetched from translation, so it's send localized according to user preference.

**Comment by pitpalme** [comment]
iCal attachment obsolete by #558?

**Comment by christophmeissner** [comment]
> iCal attachment obsolete by #558?

No, I don't think so. I would keep it in the confirmation emails as an event invitation for everybody, but we need to make sure, that the UUIDs of the events match, to prevent duplications when the user has in fact subscribed to the ics feed and accepts the attached invitations.

**Comment by Doca** [comment]
To to be clear on the unique prodid for the ical file. I would create a new property in the shift model which returns a unique prodid for the shift. Basically i would generate a string from base64(domainname+shift-id) which is easy to deal with. I don't know what the purpose of a UUID could be...

**Comment by christophmeissner** [comment]
I'm not sure about the `prodid` of an ical feed, but I think that is a property of an entire calendar, which would be sth like the URL of calendar feed (eg. `https://www.volunteer-planner.org/user/123/shifts/calendar.ics`). I was talking about event ids for individual calendar events (`VEVENT` > `UID`) which would appear in the personal calendar feed for a specific user (ie. per `user.account`) and per event (ie. per `ShiftHelper`?). A UUID is a standardized way of generating universal unique identifiers.
